rest_configuration.cpp

```diff
@@ -2173,6 +2173,29 @@ int DeRestPluginPrivate::modifyConfig(const ApiRequest &req, ApiResponse &rsp)
         rspItem["success"] = rspItemState;
         rsp.list.append(rspItem);
     }
+    if (map.contains("lightlastseeninterval")) // optional
+    {
+        int lightLastSeen = map["lightlastseeninterval"].toInt(&ok);
+        if (!ok || lightLastSeen = 0 || lightLastSeen > 65535)

```

Comment:
Shouldn't this also guard again negative values?

`if (!ok || lightLastSeen <= 0 || lightLastSeen > 65535)`

Reply:
Ah, yes. Probably better.